app/scripts/modules/core/src/search/infrastructure/SearchResult.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { NgReact } from 'core/reactShims';
+import { IRecentHistoryEntry } from 'core/history';
+
+export interface ISearchResultProps {
+  item: IRecentHistoryEntry & { account?: string; displayName?: string };
+}
+
+export class SearchResult extends React.Component<ISearchResultProps> {
+  public render() {
+    const { AccountTag } = NgReact;
+    const { item } = this.props;
+    const params = item.params || {};
+    const account: string = item.account || params.account || params.accountId || params.accountName;
```

Comment:
Do you need to explicitly declare this as a `string`?

Reply:
no